src/package/packageVersionList.ts

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as _ from 'lodash';
```

Comment:
lodash is another library I'd try to avoid bringing in as a dep if possible.

Reply:
Ok, I will work on that for my next PR.